cca/evidence.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2023 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+package cca
+
+import (
+	"crypto"
+	"crypto/sha512"
+	"errors"
+	"fmt"
+	"reflect"
+
+	"github.com/fxamacker/cbor/v2"
+	"github.com/veraison/ccatoken"
+)
+
+type Evidence struct {
+	Kat    KAT
+	Pat    ccatoken.Evidence
+	rawKAT []byte
```

Comment:
just for consistency, should we rename this to 

rawKAT cbor.RawMessage

Reply:
since this is not intended to be used directly in the context of CBOR encoding or decoding it makes little sense to use a `fxamacker/cbor` type.